File: javascript/simple/1-TwoSum.js
var twoSum = function(nums, target) {
    /* solution for n^2 */
    var arr = new Array(2);
    var NumsLength = nums.length;
    for (var i = 0; i < NumsLength; i++) {
        for (var j = i + 1; j < NumsLength; j++) {
            if (nums[i] + nums[j] === target) {
                arr[0] = i;
                arr[1] = j;
                break;
            }
        }
    }
    return arr;
};

File: java/simple/1-TwoSum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int i, j;
        int len = nums.length;
        int[] arr = new int[2];
        for (i = 0; i < len; i++) {
            for (j = i + 1; j < len; j++) {
                if (nums[i] + nums[j] == target) {
                    arr[0] = i;
                    arr[1] = j;
                    break;
                }
            }
        }
        return arr;
    }
}
